### ISSUE TITLE
Provide ready to use Mac executable

### ISSUE THREAD

**Issue by angel-git** [Canop/safecloset#19]
This tool looks great, although mac users can use Mac keychain this looks very useful to send closets to other people but there is no darwin executable on the released zip

**Comment by Canop** [comment]
@angel-git I can't compile Mac executable myself, because you need parts that, to my knowledge, aren't freely available when you don't own a mac. So, while the application is designed to compile and run on Mac, I can't directly provide the executable.

(I'd welcome a proven, legal and safe solution)

**Comment by angel-git** [comment]
fair enough 👍 i don't know how much safe is using something like this: https://wapl.es/rust/2019/02/17/rust-cross-compile-linux-to-macos.html

Feel free to close this :)  i will open an extra issue but i don't know if it's just happening on mac :)

**Comment by Canop** [comment]
I won't close the issue, it's real. But I don't think it's going to be solved soon: I need to be clean and clear on the whole process (including getting the SDK and testing) before I release a Mac executable.
